### PR TITLE
Reminder to finish your state return notifications

### PR DESCRIPTION
**Pull request by rickreyhsig**
https://www.pivotaltracker.com/story/show/186497037

### Manual testing instructions
```
require 'factory_bot'
FactoryBot.find_definitions
include FactoryBot::Syntax::Methods
intake = create(:state_file_ny_intake)
intake.update(email_address: 'rkreyhsig@codeforamerica.org', email_address_verified_at: Time.now)
StateFile::MessagingService.new(message: StateFile::AutomatedMessage::FinishReturn, intake: intake).send
_message
```

### Rake task
```
rake state_file:reminder_to_finish_state_return
```

#### English
Screen Shot 2024-03-18 at 5 54 22 PM

#### Spanish
Screen Shot 2024-03-18 at 5 58 21 PM

### CODE REVIEW

**Review by tofarr — APPROVED** [review]

File: spec/services/state_file/messaging_service_spec.rb [hunk 1]
```diff
@@ -43,4 +43,18 @@
       expect(intake.message_tracker).to eq({})
     end
   end
+
+  context "when message is an finish_return" do
```
Comment (rickreyhsig) [inline spec/services/state_file/messaging_service_spec.rb:47]:
```suggestion
  context "when message is of finish_return type" do
```

**Review by tofarr — APPROVED** [review]

File: app/services/efile/reminder_to_finish_state_return_service.rb
Comment (mpidcock) [inline app/services/efile/reminder_to_finish_state_return_service.rb:1]: I think this service belongs in /state_file, not /efile

Comment (rickreyhsig) [reply app/services/efile/reminder_to_finish_state_return_service.rb:1]: Good call, updating...

File: spec/services/state_file/reminder_to_finish_state_return_service_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        create :state_file_az_intake,
+               df_data_imported_at: (11.hours + 59.minutes).ago
+      end
+      it "does not send a message to the email associated with the intake" do
+        StateFile::ReminderToFinishStateReturnService.run
+        expect(StateFile::MessagingService).to_not have_received(:new)
+      end
+    end
+  end
+end
```
Comment (rickreyhsig) [inline spec/services/state_file/reminder_to_finish_state_return_service_spec.rb:38]:
```suggestion
end

```

**Review by mpidcock — APPROVED** [review]
📨

**Pull request merged**